Cybersecurity New Topic: Vulnerability (#1702)
Codecademy/docs: content/cybersecurity/concepts/vulnerability/vulnerability.md

---
Title: 'Vulnerability'
Description: 'Vulnerability is the weakness in an asset or the weakness of a safeguard.'
Subjects:
  - 'Computer Science'
  - 'Cybersecurity'
Tags:

>   - 'Cybersecurity'

Reviewer: Can we add more tags?

  - 'Hacking'
CatalogContent:
  - 'introduction-to-cybersecurity'
  - 'paths/cyber-resilience-risk-management'
---

A **vulnerability** is the weakness in an asset or the weakness of a safeguard. Vulnerabilities are targets for cybercriminals who often try to exploit them by gaining illegal access to systems.

> ## Vulnerability In-Use

Reviewer 2: I'm unsure if this heading is accurate to the info below. If you think differently I'd love to discuss

> Some examples of vulnerabilities include: system misconfigurations, unpatched software, poor data encryptions, weak authorization credentials. All of these are weaknesses that can be used by cybercriminals to harm an organization.

Reviewer 2: Can we take the mentioned vulnerabilities and list them in bullet point format? It would be nice to have a quick explanation for each as well.

## Identifying Vulnerabilities

> The most common way to identify and prevent vulnerabilities is through a vulnerability assessment. This essentially is a systematic review to see what, if any, vulnerabilities exist, the criticality of them, and recommendations to remediate them if needed. A vulnerability assessment will scan the network to identify any software and configurations with vulnerabilities that have not been remediated. This type of test can generally uncover things like unpatched software, network protocols using outdated encryption, or exposed ports not protected behind a firewall.

Reviewer 2: ```suggestion
The most common way to identify and prevent vulnerabilities is through a vulnerability assessment. This is a systematic review to see what, if any, vulnerabilities exist, the criticality of them, and recommendations to remediate them if needed. A vulnerability assessment will scan the network to identify any software and configurations with vulnerabilities that have not been remediated. This type of test can generally uncover things like unpatched software, network protocols using outdated encryption, or exposed ports not protected behind a firewall.
```

## Vulnerabilities vs. Threats

> Vulnerabilities and threats are often used interchangeably, however, it is important to distinguish these terms. First, a vulnerability can make an organization susceptible to threats. A threat is a malicious event that exploits a vulnerability. More specifically, a threat is when an adversary has the opportunity, capability and intent to negatively impact an organization’s people, processes or technology. Some examples of threats include malware, ransomware, and phishing attacks.

Reviewer 2: Can this be simplified:

```suggestion
A vulnerability can make an organization susceptible to threats. A threat is a malicious event that exploits a vulnerability. More specifically, a threat is when an adversary has the opportunity, capability, and intent to negatively impact an organization’s employees, processes, or technology. Some examples of threats include malware, ransomware, and phishing attacks.
```
